### PR TITLE
Add the 'a: desktop' label to labeler.yml

### PR DESCRIPTION
**Pull request by loic-sharma**
The desktop team uses the [`a: desktop`](https://github.com/flutter/engine/pulls?q=is%3Aopen+label%3A%22affects%3A+desktop%22+sort%3Aupdated-desc) label to [triage](https://github.com/flutter/flutter/wiki/Triage#desktop-platforms-team-team-desktop) pull requests.

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@
   - '**/animation/*'
   - '**/*animation*'
 
+'a: desktop':
+  - '**/linux/*'
```
Comment (CaseyHillers) [inline .github/labeler.yml:17]: Should these have an additional directory wildcard at the end?

`  - '**/linux/**/*'`

**Review by CaseyHillers — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (loic-sharma) [reply .github/labeler.yml:17]: That's what I thought too, but the other paths don't seem to follow that pattern. For example, `material` definitely has nested folders, yet its pattern is [`**/material/*](https://github.com/flutter/flutter/blob/master/.github/labeler.yml#L51). I'll add the additional directory wildcard

**Review by cbracken — APPROVED** [review]
I swear I remember lgtm-ing this, but maybe that was a different but related patch? Either way, lgtm!

Edit: Ah that was the engine side patch.

**Comment by loic-sharma** [comment]
@cbracken That was the engine repo's version of this pull request :)

**Pull request merged**